packages/paste-website/src/components/customization-landing-page/LandingPageHero.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import {graphql, useStaticQuery} from 'gatsby';
+import Img from 'gatsby-image';
+import {Box} from '@twilio-paste/box';
+import {Button} from '@twilio-paste/button';
+import {Text} from '@twilio-paste/text';
+import {Heading} from '@twilio-paste/heading';
+
+import {WorksGreatWith} from './WorksGreatWith';
+import {LandingPageSectionContent} from './LandingPageLayoutUtils';
+
+const imageQuery = graphql`
+  query {
+    sliderImage: file(sourceInstanceName: {eq: "assets"}, relativePath: {eq: "images/customization/hero-slider.png"}) {
+      childImageSharp {
+        fluid(maxWidth: 640) {
+          ...GatsbyImageSharpFluid
+        }
+      }
+    }
+    sliderImageSmall: file(
+      sourceInstanceName: {eq: "assets"}
+      relativePath: {eq: "images/customization/hero-slider-mobile.png"}
+    ) {
+      childImageSharp {
+        fluid(maxWidth: 640) {
+          ...GatsbyImageSharpFluid
+        }
+      }
+    }
+  }
+`;
+
+export const LandingPageHero: React.FC = () => {
+  const imageData = useStaticQuery(imageQuery);
+
+  return (
+    <Box>
+      <Box padding="space70">
+        <Box
+          backgroundColor="colorBackground"
+          borderRadius="borderRadius20"
+          paddingX={['space90', 'space160']}
+          paddingY={['space90', 'space190']}
+          position="relative"
+        >
+          <LandingPageSectionContent variant="default">
+            <Box maxWidth={['none', 'size70']} width={['100%', '40%']}>
+              <Heading as="div" variant="heading40">
+                <Text as="span" color="colorTextNeutral">
+                  Paste for Programmable Apps
+                </Text>
+              </Heading>
+              <Text
+                as="h1"
+                fontSize={['fontSize90', 'fontSize110']}
+                fontWeight="fontWeightBold"
+                lineHeight={['lineHeight90', 'lineHeight110']}
+                marginBottom="space70"
+              >
+                Customize your Twilio-powered apps using Paste
+              </Text>
+              <Text as="p" fontSize="fontSize60" lineHeight="lineHeight60" marginBottom="space70">
+                Rapidly build enterprise-grade, inclusive, and extensible UIs that look and feel like your brand.
+              </Text>
+              <Button as="a" href="/customization/customization-provider" variant="primary">
+                Start customizing
+              </Button>
+            </Box>
+            <Box
+              display={['none', 'block']}
+              position="absolute"
+              right="spaceNegative160"
+              top="50px"
+              maxWidth="size70"
+              width="60%"
+              zIndex="zIndex10"
+            >
+              <Img fluid={imageData.sliderImage.childImageSharp.fluid} />
```

Comment:
Oddly enough, the large image has a blurry cursor on the slider portion, but the small one doesn't. Can we remove the cursor as it's confusing?